src/app/containers/ProductListPage/ProductListPage.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import { Link } from 'react-router-dom';
+
+class ProductListPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      products: [
```

Comment:
There are no reasons to use state for products, because products never change, so please use just const